src/app/services/user-service/user.service.ts

```diff
@@ -55,25 +55,29 @@ export class UserService {
 	 * A GET method to retrieve one user from the database with the given id. Return a promise.
 	 * @param idParam 
 	 */
-	getUserById(idParam: number){
+	getUserById(idParam: number) 
+	{
```

Comment:
The opening curly bracket should be on the same line as the method declaration with a space between.